lib/misc/cells.coffee

```diff
@@ -0,0 +1,32 @@
+scopes = require './scopes'
+{values} = require 'underscore-plus'
+
+module.exports =
+
+  getCurrentCell: (ed) ->
+    buffer = ed.getBuffer()
+    start = buffer.getFirstPosition()
+    end = buffer.getEndPosition()
+    regexString = atom.config.get 'julia-client.cellDelimiter'
+
+    unless regexString?
+      return [start, end]
+
+    regex = new RegExp regexString
+    cursor = ed.getCursorBufferPosition()
```

Comment:
AFAICT the blocks code seems to iterate over selections, which I've avoided for now since it probably adds a fair bit more complexity to this change. If people want us to add that feature later, then we should look into it. I hope that's not a dealbreaker 😟 but I think this is probably useable enough as it stands without handling (multiple) selections and it's reasonably likely a fair few people probably want this feature merged, asap, pretty bad 🔥🔥🔥